src/VSCodeExtension/src/formatter/rules/space-after-if.ts

```diff
@@ -0,0 +1,5 @@
+import { FormatRule } from "../formatter";

```

Comment:
I would rename this file to something like `controls.ts` and have this as one of the `rules`. My proposal is that each file within `./rules` is a logical grouping of related rules.

Reply:
remaned to control-structures.ts